README.md

```diff
@@ -1,8 +1,16 @@
-# catalogue
+# Catalogue
 
 [![Build Status](https://travis-ci.org/wellcometrust/catalogue.svg?branch=master)](https://travis-ci.org/wellcometrust/catalogue)
 
-The Catalogue API allows you to search our museum and library collections.
+
+## Purpose
+
+Making the Wellcome Collection's data and information open and discoverable via public APIs.
```

Comment:
nit: I believe style guide is “Wellcome Collection”, not “the Wellcome Collection”.

Also, I think it might be worth explaining a little bit more about what that data actually is, and what Wellcome is (the old README did this better). Everybody has data and information – what are we exposing? Is it museum/library data? Financial records? Colour palettes? Employee data? &c.

Somebody has been linked to this repo – what does the Catalogue API contain?

Reply:
This can be linked to from the different parts of the architecture below.